functions/passing-parameters-by-value.md

```diff
@@ -0,0 +1,91 @@
+# Module name: Passing Parameters by Value
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+## Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational</td>
+    <td>Defining and calling functions with values</td>
+  </tr>
+  <tr>
+    <td>Main</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td></td>
+  </tr>
+</table>
+
+## Motivation
+
+_Why is this important?_
+_Why do we want to learn/teach this topic?_
+
+The fundamental element for code-reuse is a function. For functions to be 
+useful, we need to parameterize them. Understanding how to do such is thus
+fundamental to programming in any language.
+
+## Topic introduction
+
+_Very brief introduction to the topic._
+
+Explain how to define functions with parameters and call them with values.
+
+## Foundational: Using reference types to avoid copies
+
+### Background/Required Knowledge
+
+A student is able to:
+
+* Explain what a function is
+
+### Student outcomes
+
+_A list of things "a student should be able to" after the curriculum._
+_The next word should be an action word and testable in an exam._
+_Max 5 items._
+
+A student should be able to:
```

Comment:
I think this _is_ "function calling". But we distinguish the topic name from "calling by reference" as we expect this to be more foundational than understanding "call by reference" as the later is more complex in grammar

Reply:
Well, that depends on the meaning of *is*. The title of this topic is "passing by value". Parameter type & count agreement is more basic, and presumably will use examples where non-variable expressions are used, meaning `f(1.0,"foobar")`, just to avoid the value/reference distinction.